recipe/meta.yaml

```diff
@@ -1,45 +1,78 @@
-{% set name = "<PLEASE ADD PKG NAME>" %}
-{% set version = "<PLEASE ADD PKG VERSION>" %}
+{% set name = "raphtory" %}
+{% set version = "0.13.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: <sha256>
+  url: https://github.com/Pometry/Raphtory/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f99caa183387c68a3cdc0949c9000c23553cb3f33524989fa14170a3f522f69d
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  skip: True  # [py<38]
+  script:
+    - {{ PYTHON }} -m pip install ./python -vv --no-deps --no-build-isolation
+    - cargo-bundle-licenses --format yaml --output "{{ SRC_DIR }}\THIRDPARTY.yml"
```

Comment:
`\` or `/` -- on Unix this will become `{{ SRC_DIR }}THIRDPATY.yml` which might be right or might not.

Reply:
Fixed in [369760e](https://github.com/AnacondaRecipes/raphtory-feedstock/pull/1/commits/369760e4878b886a28a2f42d1684e64f5f080507).